bridge.js

```diff
@@ -27,32 +27,31 @@ function RasBridge (getRas) {
   }
 }
 
-function propagate(data, ras) {
-  switch (data.action) {
+function propagate (request, ras) {
+  switch (request.action) {
     case Action.OPEN:
-      ras._open(data)
+      ras.open(request.callback)
       return
     case Action.OPENREADONLY:
-      ras._openReadonly(data)
+      ras._openReadonly(request)
```

Comment:
@RangerMauve I've kept only this one, WDYT?

Reply:
@soyuka Maybe check that it exists first and invoke `open` if it doesn't.